monai/data/meta_tensor.py

```diff
@@ -572,14 +572,14 @@ def __repr__(self):
         Prints a representation of the tensor identical to ``torch.Tensor.__repr__``.
         Use ``print_verbose`` for associated metadata.
         """
-        return self.as_tensor().__repr__()
+        return "meta" + self.as_tensor().__repr__()
 
     def __str__(self):
         """
         Prints a representation of the tensor identical to ``torch.Tensor.__str__``.
         Use ``print_verbose`` for associated metadata.
         """
-        return str(self.as_tensor())
+        return "meta" + str(self.as_tensor())
```

Comment:
I think this is good idea, could you please make a dedicated PR for this feature?

Reply:
I've taken these changes out of this fork, to reapply in another PR!